lib/screens/create_region_screens/add_region_description/add_region_description.dart

```diff
@@ -39,9 +63,14 @@ class AddRegionDescription extends StatelessWidget {
                           child: Column(
                             children: [
                               TextFormFieldCustom(
-                                  autofocus: true,
-                                  maxLines: 10,
-                                  labelText: context.loc.createRegionAddDescriptionInputFormTitle),
+                                controller: _keyController,
+                                autofocus: true,
+                                maxLines: 10,
+                                labelText: context.loc.createRegionAddDescriptionInputFormTitle,
+                                onChanged: (text) {
+                                  BlocProvider.of<CreateRegionBloc>(context).add(OnRegionDescriptionChange(text));
+                                },
```

Comment:
If you going to use the controller you do not need the onChange that is the reason on the listener in the initial state.